src/js/common/schemaform/review/ReviewPage.jsx

```diff
@@ -54,22 +57,28 @@ export class ReviewPage extends React.Component {
   }
 }
 
-function mapStateToProps(state, ownProps) {
+function mapStateToProps(state) {
   return {
-    form: state.form,
-    formConfig: ownProps.route.formConfig
+    form: state.form
   };
 }
 
 const mapDispatchToProps = {
   setEditMode,
   setSubmission,
   submitForm,
-  setPrivacyAgreement
+  setPrivacyAgreement,
+  setData,
+  setValid
 };
 
 ReviewPage.propTypes = {
-  formConfig: React.PropTypes.object.isRequired
+  form: React.PropTypes.object.isRequired,
+  route: React.PropTypes.shape({
+    formConfig: React.PropTypes.object.isRequired
+  }).isRequired
 };
 
 export default connect(mapStateToProps, mapDispatchToProps)(ReviewPage);
+
+export { ReviewPage };
```

Comment:
Typically I've been doing the export on the class declaration rather than the separate export statement, but either way works.